2-library/6core-logging/src/main/java/com/_6core/lib/java/MicroserviceLogger.java

```diff
@@ -0,0 +1,25 @@
+package com._6core.lib.java;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public interface MicroserviceLogger {
+
```

Comment:
Why do we need this interface? We will never use it, because logger should be instantiated on class level, e.g. 

getLoggerFactory(MyClass.class) to see in log which class is acting now.